crates/fuel-core/src/schema/block.rs

```diff
@@ -277,21 +276,13 @@ where
 #[derive(Default)]
 pub struct BlockMutation;
 
-#[derive(InputObject)]
-struct TimeParameters {
-    /// The time to set on the first block
-    start_time: U64,
-    /// The time interval between subsequent blocks
-    block_time_interval: U64,
-}
-
 #[Object]
 impl BlockMutation {
     async fn produce_blocks(
         &self,
         ctx: &Context<'_>,
+        start_timestamp: Option<U64>,
```

Comment:
It would be nice to have a doc comment indicating start timestamp is in Tai64 format, as it's not as common and may cause some confusion if information is not preserved upstream